crates/biome_service/src/file_handlers/mod.rs

```diff
@@ -525,6 +527,8 @@ pub(crate) struct Features {
     unknown: UnknownFileHandler,
     graphql: GraphqlFileHandler,
     html: HtmlFileHandler,
+    #[allow(unused)]
```

Comment:
nit: this is new in rust 1.81
```suggestion
    #[expect(unused)]
```